lib/request.ts

```diff
@@ -84,7 +84,7 @@ export default class CIORequest {
 
         res.on('end', () => {
           let body = Buffer.concat(chunks).toString('utf-8');
-          let json = null;
+          let json = {};
```

Comment:
This might still end up getting inferred as a `{}` type, so it might be better to do:

```typescript
          let json: Record<string, any> = {};
```